app/javascript/src/Navigation/utils/getActiveMenuTitle.jsx

```diff
@@ -0,0 +1,32 @@
+export default function getActiveMenuTitle (activeMenu, currentApi) {
+  switch (activeMenu) {
+    case 'dashboard':
+      return 'Dashboard'
+
+    case 'personal':
+      return 'Personal Settings'
+
+    case 'account':
+      return 'Account'
+
+    case 'buyers':
+    case 'finance':
+    case 'cms':
+    case 'site':
+      return 'Audience'
+
+    case 'settings':
+      return 'Preferences'
+
+    case 'applications':
+    case 'active_docs':
+      return 'All APIs'
+
+    case 'serviceadmin':
+    case 'monitoring':
+      return `API: ${currentApi.service.name}`
+
+    default:
```

Comment:
it could be useful to have default "Dashboard", and write a simple test with some of the options. Also, next iteration applying flow types and define `currentApi`, since we are relying on it calling 2 methods.

Reply:
How about _Choose an API_ as default case?